docs/react-v9/contributing/migration-notes.md

```diff
@@ -0,0 +1,407 @@
+# Migration notes from `@fluentui/react-components@9.0.0-beta.5`to `@fluentui/react-components@9.0.0-rc.1`
```

Comment:
Not sure that we want to this document in a new location. It was good in its time, some info in it is already outdated.

Reply:
Yeah agreed. The shorthands info is definitely out of date.

Reply:
Removed this doc from the PR.